core/modules/query/ValueExpr.cc

```diff
@@ -271,17 +271,17 @@ std::ostream& operator<<(std::ostream& os, ValueExpr const& ve) {
 }
 
 std::ostream& operator<<(std::ostream& os, ValueExpr const* ve) {
-    if(!ve) return os << "<NULL>";
+    if (!ve) return os << "<NULL>";
     return os << *ve;
 }
```

Comment:
But ValueExpr does not have operator?, so going that route would be more trouble than it is worth it.